### PR TITLE
Add unlearnable attribute to spells via fixes

### PR DESCRIPTION
**Pull request by sigilbaram**
* Allows addons to filter out unlearnable spells
* make booleans parse in and export properly

### CODE REVIEW

**Comment by sigilbaram** [comment]
Not used in any addons, so it changing bags.lua doesn't matter.

File: ExtensionMethods.cs [hunk 1]
```diff
@@ -199,6 +199,12 @@ public static object Parse(this XAttribute value)
                 return resfloat;
             }
 
+            bool resbool;
+            if (bool.TryParse(str, out resbool))
+            {
+              return resbool;
```
Comment (z16) [inline ExtensionMethods.cs]: The spacing is off, please use 4 space indentation.

File: ExtensionMethods.cs [hunk 2]
```diff
@@ -1,16 +1,16 @@
 ﻿// <copyright file="ExtensionMethods.cs" company="Windower Team">
 // Copyright © 2013-2014 Windower Team
```
Comment (z16) [inline ExtensionMethods.cs]: May wanna update the copyright to 2018, since you just changed the file.

File: Serializers/Lua/LuaAttribute.cs [hunk 3]
```diff
@@ -1,16 +1,16 @@
 ﻿// <copyright file="LuaAttribute.cs" company="Windower Team">
 // Copyright © 2013-2017 Windower Team
```
Comment (z16) [inline Serializers/Lua/LuaAttribute.cs]: Same as above.

File: Serializers/Lua/LuaAttribute.cs [hunk 4]
```diff
@@ -57,6 +57,11 @@ private static string MakeValue(object value)
                 return "\"" + value.ToString().Replace("\"", "\\\"").Replace("\n", "\\n") + "\"";
             }
 
+            if ( value is bool )
+            {
+              return value.ToString().ToLower();
```
Comment (z16) [inline Serializers/Lua/LuaAttribute.cs]: Same as above, and the spaces inside the parens.

Comment (sigilbaram) [reply Serializers/Lua/LuaAttribute.cs]: 2f2697d, 3cf6503

Not sure what I did there that double committed and then did a branch merge... Didn't think git would even commit something that was a non-change... >.<

**Pull request merged**